content/reference/sql/supported-clients/sql-editors.md

```diff
@@ -7,40 +7,101 @@ title: SQL Editors
 Dolt comes with a built in MySQL compatible server that you can connect SQL Editors too. Here are a list of notable MySQL Editors and our compatibility status.
 | Editor | Supported | Notes and limitations |
 | :--- | :--- | :--- |
-| [Tableplus](https://tableplus.com/) | ✅ | |
+| [Tableplus](https://tableplus.com/) | ✅ | Recommended (see below) |
 | [Navicat For Mysql](https://www.navicat.com/en/products/navicat-for-mysql) | ✅ | |
 | [Dbeaver](https://dbeaver.io/) | ✅ | |
-| [Datagrip](https://www.jetbrains.com/datagrip/) | ✅ | Need to implement a couple functions |
+| [Datagrip](https://www.jetbrains.com/datagrip/) | ✅ | |
 | [MySQL Workbench](https://www.mysql.com/products/workbench/) | ❌ | Missing some information schema materials |
 
-## Recommended Dolt Configuration
+## Setting up Dolt with Tableplus
 
-Some editors require custom configuration of dolt's sql-server. For example, DBeaver use multiple connections to run queries whereas `dolt sql-server` supports 1 connection out of the box. Here's a sample config file that should get you started with `dolt sql-server`
+[Tableplus](https://tableplus.com/) is the recommended SQL editor to use with dolt. Let's clone a Dolt repo and connect it to Tableplus.
 
-<div class="gatsby-highlight" data-language="text">
-	<pre class="By default, starts a MySQL-compatible server whilanguage-text">
-		<code class="language-text">
+### Step 1: Cloning a repository.
 
-log_level: trace
+Open your terminal and clone the following [repository](https://www.dolthub.com/repositories/dolthub/ip-to-country/data/master).
```

Comment:
(minor) Should we try to consistently use "database" instead of repository here? It sounded like we were hearing that term resonate better with customers?

Reply:
Yes, always say database